traja/models/nn.py

```diff
@@ -122,10 +240,11 @@ def __init__(self, model,
 
         self.train_loader = train_loader
         self.test_loader = test_loader
-
         self.warmup_steps = warmup_steps
-
-        self.criterion = LossMseWarmup(self.warmup_steps)
+        if self.warmup_steps!=None: 
+            self.criterion = LossMseWarmup(self.warmup_steps)
+        else:
+            self.criterion = LossMse(self.warmup_steps) # warmup_steps == None
```

Comment:
Why accept warmup steps if we don't use them?

Reply:
It is just a check to have backward compatibility with other models. There are few models in  nn.py which are still not tested. Currently, we are using only LSTMs. But yes, I also strongly discourage warmup steps. We could use warm steps feature inside the optimizer. For  that, I will create an issue instead and assign myself to address it.

Reply:
#24  is the issue

Reply:
Added user-defined trial dir name, removed the warmup steps and correponding loss functions. In the future, a warmup scheduler will be added.